frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -45,11 +45,21 @@ export function OperatorValueSelect({
     operatorSelectProps,
     allowQueryingEventsByDateTime,
 }: OperatorValueSelectProps): JSX.Element {
-    const [currentOperator, setCurrentOperator] = useState(operator)
     const { propertyDefinitions } = useValues(propertyDefinitionsModel)
 
     const propertyDefinition = propertyDefinitions.find((pd) => pd.name === propkey)
 
+    // DateTime properties should not default to Exact
+    const startingOperator =
+        allowQueryingEventsByDateTime &&
+        propertyDefinition?.property_type == 'DateTime' &&
+        (!operator || operator == PropertyOperator.Exact)
+            ? PropertyOperator.IsDateBefore
+            : operator
+            ? operator
+            : PropertyOperator.Exact
```

Comment:
This is trippy and hard to follow 🤪. Could `: (operator || PropertyOperator.Exact)` make anything cleaner instead of another `?:`?

Reply:
It wasn't _too_ bad before Prettier reformated it... but, yep